pkg/generate/app/componentresolvers.go

```diff
@@ -41,6 +41,12 @@ func (r PerfectMatchWeightedResolver) Resolve(value string) (*ComponentMatch, er
 	imperfect := ScoredComponentMatches{}
 	group := []WeightedResolver{}
 	for i, resolver := range r {
+		if isFile(value){
+			_, err := resolver.Searcher.Search(value)
```

Comment:
if _, err := resolver.Searcher.Search(value); err != nil {
  return nil, err
}

(since you're not using the error anywhere below).